pom.xml

```diff
@@ -49,7 +49,7 @@
         <vault.user>admin</vault.user>
         <vault.password>admin</vault.password>
         <core.wcm.components.version>2.10.0</core.wcm.components.version>
-        <core.cif.components.version>1.4.0</core.cif.components.version>
+        <core.cif.components.version>1.4.1-SNAPSHOT</core.cif.components.version>
```

Comment:
Don't forget to update the version after cif components package supporting data layer is released